lib/load.php

```diff
@@ -44,8 +44,16 @@ function gutenberg_is_experiment_enabled( $name ) {
 	// WordPress 6.8 compat.
 	require __DIR__ . '/compat/wordpress-6.8/rest-api.php';
 
+	// WordPress 7.0 compat
+	/*
+	 * These 7.0 compat classes are used to extend the templates controller.
+	 * They are placed before the 6.9 compat classes so that they can be used to
+	 * extend the templates activation controller.
+	 */
+	require __DIR__ . '/compat/wordpress-7.0/class-gutenberg-rest-templates-controller.php';
+	require __DIR__ . '/compat/wordpress-7.0/class-gutenberg-rest-registered-templates-controller.php';
```

Comment:
Added before the template activation code so it can inherit. 

I started moving things around but it just got messy. Or messier.